setup.py

```diff
@@ -34,14 +34,14 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='edx-proctoring',
-    version='0.13.0',
+    version='0.14.0',
     description='Proctoring subsystem for Open edX',
     long_description=open('README.md').read(),
     author='edX',
     url='https://github.com/edx/edx-proctoring',
     license='AGPL',
     classifiers=[
-        'Development Status :: 3 - Alpha',
+        'Development Status :: 4 - Beta',
```

Comment:
Good question. I found the list of valid classifiers here:

https://pypi.python.org/pypi?%3Aaction=list_classifiers

My thinking is that there is no way this is an alpha since we're using it in production. I was tempted to go with ``Development Status :: 5 - Production/Stable`` but then figured that should probably wait until we fix the polling issues. At that point I think we should switch to version 1.0.0 and mark it as production/stable. 

Does that make sense?